src/components/ConditionalFilter/filterDefinitions/definitions/AttributeDefinition.ts

```diff
@@ -0,0 +1,160 @@
+import { ApolloClient } from "@apollo/client";
+import { AttributeInput, AttributeInputTypeEnum } from "@dashboard/graphql";
+
+import { AttributeChoicesHandler, Handler } from "../../API/Handler";
+import { FilterElement } from "../../FilterElement";
+import { WhereOnlyFilterDefinition } from "../types";
+import { extractConditionValueFromFilterElement, getBooleanValueFromElement } from "../utils";
+
+export class AttributeDefinition
```

Comment:
as we discuss it's probably less confising to call it like AtrtributeQueryVarsBuilder